enterprise/frontend/src/embedding-sdk/hooks/private/get-auth-configuration.ts

```diff
@@ -52,10 +47,6 @@ export const getAuthConfiguration = (
       jwtProviderUri => setupJwtAuth(jwtProviderUri, dispatch),
     )
     .with([{ apiKey: P.select(P.nonNullable) }, "localhost"], apiKey => {
-      presentApiKeyUsageWarning(appName);
       setupLocalApiKey(dispatch, apiKey);
     })
-    .with([{ apiKey: P.select(P.nonNullable) }, P.not("localhost")], () =>
-      getErrorMessage("PROD_API_KEY"),
-    )
```

Comment:
Both the logic for presenting the API key warnings and for showing errors when using API keys in production are part of the new code, so those are removed from `getAuthConfiguration`.